### PR TITLE
Tooltips on disabled buttons in navbar switch

### LINKED ISSUE #1820: Navbar switch button tooltips
The language buttons (EN/DE) and mode buttons (regular mode/staff mode) in the navbar should have styled bootstrap tooltips. The tooltips should be shown for enabled and disabled buttons.
Currently, only the enabled mode button has a correctly styled tooltip.

### PR DESCRIPTION
**Pull request by tanja04**
closes #1820

### CODE REVIEW

**Review by niklasmohrin — APPROVED** [review]
Nice solution, thanks :)

**Review by He3lixxx — APPROVED** [review]
Nice, thank you :)

**Review by janno42 — CHANGES_REQUESTED** [review]
I'm gonna be a bit picky here. The current solution breaks the button group layout:
image

It should instead have rounded edges only at the outmost corners and not in between:
image

Please have a look into the styling :)

File: evap/static/scss/_utilities.scss
Comment (niklasmohrin) [inline evap/static/scss/_utilities.scss:1]: Please revert the changes on this file (we can show you in the next meeting)

**Review by niklasmohrin — APPROVED** [review]
Very nice :) One final thing:

File: evap/evaluation/templates/navbar.html [hunk 1]
```diff
@@ -104,13 +104,13 @@
...
                                     <span id="span-exit-staff-mode" class="fas fa-user"></span>
                                 </button>
                             </form>
                             <form id="enter-staff-mode-form" class="d-flex" method="post" action="{% url 'staff:enter_staff_mode' %}">
                                 {% csrf_token %}
-                                <button type="submit" {% if user.is_staff %}class="btn btn-sm btn-navbar active" disabled{% else %}class="btn btn-sm btn-navbar"{% endif %} data-bs-toggle="tooltip" data-bs-placement="bottom" title="{% trans 'Staff mode' %}" onclick="setSpinnerIcon('span-enter-staff-mode')">
+                                <button data-bs-toggle="tooltip" data-bs-placement="bottom" title="{% trans 'Staff mode' %}" type="submit" {% if user.is_staff %}class="btn btn-sm btn-navbar active"{% else %} class="btn btn-sm btn-navbar"{% endif %} onclick="setSpinnerIcon('span-enter-staff-mode')">
```
Comment (He3lixxx) [inline evap/evaluation/templates/navbar.html:113]: Very minor thing: The classes `btn btn-sm btn-navbar` seem to be shared between the `if` and the `else` case, they could be moved out of the conditional code (like you did in line 94).

**Review by janno42 — APPROVED** [review]
✔️ Meets requirements
✔️ UI functionality checked

**Comment by janno42** [comment]
Very nice, thanks! 🎆

**Pull request merged**